ParselyTracker/ParselyTracker.swift

```diff
@@ -177,30 +177,36 @@ public class Parsely {
     
     internal func startFlushTimer() {
         os_log("Flush timer starting", log: OSLog.tracker, type:.debug)
-        if self.flushTimer == nil && self.active {
-            self.flushTimer = Timer.scheduledTimer(timeInterval: self.flushInterval, target: self, selector: #selector(self.flush), userInfo: nil, repeats: true)
+        if flushTimer == nil && active {
+            flushTimer = Timer.scheduledTimer(timeInterval: flushInterval, target: self, selector: #selector(flush), userInfo: nil, repeats: true)
             os_log("Flush timer started", log: OSLog.tracker, type:.debug)
         }
     }
     
     internal func pauseFlushTimer() {
         os_log("Flush timer stopping", log: OSLog.tracker, type:.debug)
-        if self.flushTimer != nil && !self.active {
-            self.flushTimer!.invalidate()
-            self.flushTimer = nil
+        if flushTimer != nil && !active {
+            flushTimer!.invalidate()
+            flushTimer = nil
             os_log("Flush timer stopped", log: OSLog.tracker, type:.debug)
         }
     }
-    
+
     private func addApplicationObservers() {
         NotificationCenter.default.addObserver(self, selector: #selector(resumeExecution), name: UIApplication.didBecomeActiveNotification, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(resumeExecution), name: UIApplication.willEnterForegroundNotification, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(suspendExecution), name: UIApplication.willResignActiveNotification, object: nil)
-        NotificationCenter.default.addObserver(self, selector: #selector(suspendExecution), name: UIApplication.didEnterBackgroundNotification, object: nil)
+
+        if #available(iOS 13.0, *) {
+            NotificationCenter.default.addObserver(self, selector: #selector(suspendExecution), name: UIScene.didEnterBackgroundNotification, object: nil)
+        } else{
```

Comment:
I see you've changed the minimum deployment target to iOS 13, which means you don't need this if statement all together. You can just keep the else part 👍

Reply:
Since this is a duplicate to @aerych's comment here https://github.com/Parsely/AnalyticsSDK-iOS/pull/70#discussion_r1040093946, I'll mark this as resolved 👍